package.json

```diff
@@ -16,15 +16,16 @@
     "changeset:release": "pnpm run build && pnpm exec changeset publish",
     "changeset:version": "pnpm exec changeset version && pnpm run constellation-tracker",
     "clean": "pnpm run clean:dist && pnpm run clean:buildinfo && pnpm run clean:turbo && pnpm run clean:pnpm && pnpm run clean:cov && pnpm run clean:deps",
+    "clean:buildinfo": "pnpm exec find . -name 'tsconfig.tsbuildinfo' -delete",
     "clean:cov": "pnpm -r exec rimraf coverage && exec pnpm rimraf coverage",
     "clean:deps": "pnpm -r exec rimraf node_modules && pnpm exec rimraf node_modules",
     "clean:dist": "pnpm -r exec rimraf dist",
-    "clean:buildinfo": "pnpm exec find . -name 'tsconfig.tsbuildinfo' -delete",
     "clean:pnpm": "pnpm rimraf pnpm-lock.yaml",
     "clean:turbo": "pnpm -r exec  rimraf .turbo && exec pnpm rimraf .turbo",
     "constellation-tracker": "turbo constellation-tracker",
     "deps:format-packages": "pnpm syncpack format",
     "deps:mismatch-fix": "pnpm syncpack fix-mismatches",
+    "deps:postinstall": "pnpm run deps:mismatch-fix && pnpm run deps:format-packages",
```

Comment:
Note: in practice, this is **NOT** currently working. We will have to circle back to find out why this is the case. The idea was that once someone adds or updates a package in one of the projects, the `deps:postinstall` script is run to fix any mismatches, and then to format the `package.json` file. 

I think this deserves to be in the MR pipeline, and if anything changes, cause the pipeline to fail. That will cause the developer to look into the root cause and ask the questions.